packages/bpk-component-calendar/src/BpkCalendarDate.module.scss

```diff
@@ -75,7 +75,6 @@
   }
 
   &--middle {
-    background-color: $bpk-surface-highlight-day;
```

Comment:
It might be worth here keeping this but changing the colour, just because the `BpkCalendarDate` can be used alone alongside other custom parts of the calendar so removing this would cause a side effect of being broken with custom calendars

Reply:
Done! :)